src/plugins/translation-ui-router.js

```diff
@@ -0,0 +1,80 @@
+const _ = require('lodash');
+const { createFilter } = require('rollup-pluginutils');
+const { walk } = require('estree-walker');
+const MagicString = require('magic-string');
+const utils = require('./translation-utils');
+
+const removeProperty = (code, magicString, start, end) => {
+  magicString.remove(start, end);
+  for (let i = end; i < code.length - 1; i += 1) {
+    const ch = code[i];
+    if (ch === ',') {
+      magicString.remove(i, i + 1);
+      break;
+    } else if (/\w|}/.test(ch)) {
+      break;
+    }
+  }
+};
+
+module.exports = (opts = {}) => {
+  const include = opts.include || '**/*.js';
+  const { exclude } = opts;
+  const filter = createFilter(include, exclude);
+  const sourcemap = opts.sourcemap !== false;
```

Comment:
`sourcemapIsProvided` ? :)

Reply:
i read many rollup plugins and they all use this convention